articles/machine-learning/reference-yaml-mltable.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'CLI (v2) mltable YAML schema'
+titleSuffix: Azure Machine Learning
+description: Reference documentation for the CLI (v2) mltable YAML schema.
+services: machine-learning
+ms.service: machine-learning
+ms.subservice: mldata
+ms.topic: reference
+ms.custom: cliv2, event-tier1-build-2022
+
+author: xunwan
+ms.author: xunwan
+ms.date: 09/15/2022
+ms.reviewer: franksolomon
+---
+
+# CLI (v2) data YAML schema
```

Comment:
Shouldn't this be "CLI (v2) MLTable YAML schema" ?

Reply:
Updated.